spec/lib/secure_headers/headers/content_security_policy_spec.rb

```diff
@@ -86,8 +86,8 @@ module SecureHeaders
         expect(csp.value).to eq("default-src example.org")
       end
 
-      it "emits a warning when using frame-src" do
-        expect(Kernel).to receive(:warn).with(/:frame_src is deprecated, use :child_src instead./)
+      it "does not emit a warning when using frame-src" do
+        expect(Kernel).to_not receive(:warn).with(/:frame_src is deprecated, use :child_src instead./)
```

Comment:
This is asserting that a _particular_ warning isn't logged; we probably want to assert that _no_ warnings are logged:

```ruby
expect(Kernel).to_not receive(:warn)
```